pkg/frontend/validate.go

```diff
@@ -118,3 +118,22 @@ func validateAdminVMName(vmName string) error {
 
 	return nil
 }
+
+func validateAdminKubernetesPodLogs(method, name, namespace, containerName string) error {
+	if method != http.MethodGet {
+		return api.NewCloudError(http.StatusBadRequest, api.CloudErrorCodeInvalidParameter, "", "The provided HTTP method '%s' is invalid.", method)
+	}
```

Comment:
I don't believe you have to validate the method here because there is no write happening to kubernetes pods at all.  It will always be a `GET` action.

Reply:
Yes I agree :D